config/settings/base.py

```diff
@@ -39,10 +39,13 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'corsheaders',
     'presqt',
+    'rest_framework',
```

Comment:
Do you need to install api_v1?

Reply:
I don't think so since it's contained in '/presqt'. We can talk about these being separate apps or having the settings file install the apps inside '/presqt' more explicitly.